.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Zopen Community Discussions
+    url: https://github.com/IBM/zos-open-tools/discussions
```

Is this right?

---

Good catch, I'll fix that

```suggestion
    url: https://github.com/IBM/zopen-community/discussions
```